pkg/config/parameters.go

```diff
@@ -463,6 +471,10 @@ func (t TimeoutParameters) Validate() error {
 		return fmt.Errorf("connection shutdown grace period %q: %w", t.ConnectionShutdownGracePeriod, err)
 	}
 
+	if err := v(t.ConnectTimeout); err != nil {
```

Comment:
using `v()` here will allow `infinity` as a valid timeout, which we probably don't want, as you mentioned elsewhere

Reply:
Missed that, thanks @sunjayBhatia! Now fixed.